tests/class-wp-test-rest-post-type-controller-testcase.php

```diff
@@ -153,6 +153,48 @@ protected function check_post_data( $post, $data, $context ) {
 			sort( $data[ $taxonomy->rest_base ] );
 			$this->assertEquals( $terms, $data[ $taxonomy->rest_base ] );
 		}
+
+		// test links
+		if ( $links ) {
```

Comment:
... meaning, is there some way we can always test them? I'm not a huge fan of conditionals inside of unit tests because I think it can be easy to mess up some of your logic and not notice that your tests are no longer running.